game_view.cpp

```diff
@@ -278,19 +278,20 @@ void game_view::draw_player_coords() noexcept
 {
     sf::Text text;
     text.setFont(m_game_resources.get_font());
+    text.setCharacterSize(24);
 
     // Concatenate player coordinates string
     std::vector<player> v_player = m_game.get_v_player();
     std::string str_player_coords;
     for(int i = 0; i != static_cast<int>(v_player.size()); i++) {
         player p = v_player[static_cast<unsigned int>(i)];
-        str_player_coords += "Player " + p.get_ID() + " x = " + std::to_string(trunc(get_x(p)));
-        str_player_coords += "\nPlayer " + p.get_ID() + " y = " + std::to_string(trunc(get_y(p)));
+        str_player_coords += "Player " + p.get_ID() + " x = " + std::to_string((int)get_x(p));
```

Comment:
I understand the thought process here and I could agree with a cast to int over a `trunc`.

However, the C++ Coreguideline [ES.49: If you must use a cast, use a named cast](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es49-if-you-must-use-a-cast-use-a-named-cast) disagrees with you here. Due to this, I will ask you to improve this, as this helps you learn those C++ details better :-)

The reason why `(int)x` is a worse idea than `static_cast<int>(x)` is exactly because the latter is longer and ugly: usually casts are to be avoided.

Change this either back to `trunct` or use `static_cast<int>` and I will happily accept :+1:

Reply:
since trunc() does not work as intended, I would prefer to use static_cast<int> here, and thanks for the knowledge

Reply:
That is indeed the wiser choice :+1: